src/sass/fonts.scss

```diff
@@ -0,0 +1,86 @@
+$font-path: '../assets/fonts';
+
+@font-face {
+  font-family: 'Roboto Mono';
+  font-style: normal;
+  font-weight: 400;
+  src: url($font-path+'/Roboto-Mono-Regular.woff2') format('woff2'),
+    url($font-path+'/Roboto-Mono-Regular.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: 400;
+  src: url($font-path+'/Lato-Regular.woff2') format('woff2'),
+    url($font-path+'/Lato-Regular.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: 300;
+  src: url($font-path+'/Lato-Light.woff2') format('woff2'),
+    url($font-path+'/Lato-Light.woff') format('woff');
+}
+
+/* --------------------------------
+
+icons Web Font - nucleoapp.com/
+License - nucleoapp.com/license/
+
+-------------------------------- */
+
+@font-face {
+  font-family: 'icons';
+  src: url($font-path+'/icons.woff2') format('woff2'), url($font-path+'/icons.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
```

Comment:
Given that we use SVG files for the other icons, I'm kind of bearish on nucleo since I don't think anyone else on the team has a license, making it difficult to maintain.

Reply:
I don't see benefits of it beyond familiarity, so let's switch to formats that all on the team will be able to access.